git/README-es.md

```diff
@@ -189,12 +217,12 @@ Esta branch es creada para trabajar en el feedback de una funcionalidad.
 
 _Ejemplo: feedback/sign-up-change-inputs-order_
 
-* **Origen:** release branch
-* **Destino:** release branch
+* **Origen:** *release branch*
+* **Destino:** *release branch*
 * **Casos de uso:** Feedback del cliente o interno de una determinada funcionalidad luego de liberada una release branch a un ambiente de prueba.
 
 
-### Hotfix (hotfix/sign-up) **[Required]**
+### Hotfix (hotfix/sign-up) **[Requerida]**
```

Comment:
We should define an automatic build to QA (in case of having QA) to test the hot fix before sending the fix to prod?

Reply:
That's a good question. Taking into consideration the urgency behind a hot fix, I would say that after running the tests with CI the fix is ready to be deployed to production. After the developer merges the hotfix into develop the flow will continue as usual and a build with this fix will be created for the QA team to test. I don’t know what the rest think about this?

Reply:
I guess this is an issue of temperature :) 
If the issue is hot enough that Prod usability is impaired, I guess we could push into Prod at the same time as Staging/QA. Issues detected in these environments can always trigger a new hotfix if needed.

If we can afford to wait a day and there's potential risk of the fix breaking Prod worse than it is, we should either:
- not merge directly into master (recommended)
- merge but not deploy automatically
I'd to the first option, as long as testing in QA/Staging and subsequent marge can be performed fast enough (which would usually be the case).

Maybe leave it as a recommendation but allow for flexibility? only strict requirement being that any changes going into master are present in develop as well